### DISCUSSION TITLE
请问我部署成功后，Sonic web端一直显示后台准备中..请稍后...是我的虚拟机配置低了么

### DISCUSSION THREAD

**Discussion by Beyondbear** [SonicCloudOrg/sonic-server#423] [Q&A]

**Comment by ZhouYixun** [comment]
可以看看日志

**Comment by Beyondbear** [comment]
感谢您的回复，已经自行解决了




Beyond.bear
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SonicCloudOrg/sonic-server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月27日(星期二) 上午10:30
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [SonicCloudOrg/sonic-server] 请问我部署成功后，Sonic web端一直显示后台准备中..请稍后...是我的虚拟机配置低了么 (Discussion #423)





 
可以看看日志
